components/esp_mqtt_cxx/examples/ssl/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+# The following four lines of boilerplate have to be in your project's
+# CMakeLists in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.16)
+
+# (Not part of the boilerplate) This example uses an extra component for common
+# functions such as Wi-Fi and Ethernet connection.
```

Comment:
Maybe update this comment, that we're not only including the common-connect, but also *"this"* version of the component